src/dpp/cluster.cpp

```diff
@@ -1219,6 +1219,97 @@ void cluster::current_user_leave_guild(snowflake guild_id, command_completion_ev
 	 });
 }
 
+void cluster::current_user_join_thread(snowflake thread_id, command_completion_event_t callback) {
+	this->post_rest("/api/v9/channels",thread_id, "/thread-members/@me", m_put, "", [callback](json &j, const http_request_completion_t& http {
+		if (callback) {
+			callback(confirmation_callback_t("confirmation", confirmation(), http));
```

Comment:
do any of these return the thread info instead? or just 204 empty content? if they return any meainingful content we should find a way to pass that content to the user

Reply:
Returns a 204 empty response on success.